src/test/java/bio/terra/service/resourcemanagement/google/ResourceServiceUnitTest.java

```diff
@@ -0,0 +1,121 @@
+package bio.terra.service.resourcemanagement.google;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import bio.terra.app.configuration.SamConfiguration;
+import bio.terra.app.model.GoogleCloudResource;
+import bio.terra.app.model.GoogleRegion;
+import bio.terra.common.category.Unit;
+import bio.terra.model.BillingProfileModel;
+import bio.terra.service.dataset.Dataset;
+import bio.terra.service.dataset.DatasetBucketDao;
+import bio.terra.service.dataset.DatasetSummary;
+import bio.terra.service.dataset.StorageResource;
+import bio.terra.service.resourcemanagement.OneProjectPerProfileIdSelector;
+import bio.terra.service.resourcemanagement.ResourceService;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@Category(Unit.class)
+public class ResourceServiceUnitTest {
+
+    @InjectMocks
+    private ResourceService resourceService;
+
+    @Mock
+    private GoogleProjectService googleProjectService;
+
+    @Mock
+    private DatasetBucketDao datasetBucketDao;
+
+    @Mock
+    private OneProjectPerProfileIdSelector oneProjectPerProfileIdSelector;
+
+    @Mock
+    private GoogleBucketService bucketService;
+
+    @Mock
+    private GoogleResourceDao resourceDao;
+
+    @Mock
+    private SamConfiguration samConfiguration;
+
+    private final UUID billingProfileId = UUID.randomUUID();
+    private final BillingProfileModel profileModel = new BillingProfileModel()
+        .id(billingProfileId.toString());
+
+    private final UUID datasetId = UUID.randomUUID();
+
+    private final DatasetSummary datasetSummary = new DatasetSummary().storage(Collections
+        .singletonList(
+            new StorageResource().region(GoogleRegion.DEFAULT_GOOGLE_REGION).cloudResource(
+                GoogleCloudResource.BUCKET)));
+    private final Dataset dataset = new Dataset(datasetSummary).id(datasetId);
+
+    private final GoogleProjectResource projectResource = new GoogleProjectResource()
+        .profileId(billingProfileId);
+
+    private final UUID bucketId = UUID.randomUUID();
+    private final GoogleBucketResource bucketResource = new GoogleBucketResource()
+        .resourceId(bucketId)
+        .name("bucketName")
+        .projectResource(projectResource);
+
+    private final List<UUID> bucketsForDataset = Collections.singletonList(bucketId);
```

Comment:
Not sure about pros/cons but this can now be done using `List.of()`
```suggestion
    private final List<UUID> bucketsForDataset = List.of(bucketId);
```